### PR TITLE
DeveloperGuide: Added and explained two features

### PR DESCRIPTION
**Pull request by Sheikh-Umar**
Edited the User Story on finding duplicates to 3 more-detailed user stories on identifying a duplicate lead.

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -385,6 +385,36 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g App name, logging level) through the configuration file (default: `config.json`).
 
+=== Alias Commands feature [Currently for add, list, clear, and edit]
+
+==== Current Implementation
+
+A command can be performed by typing in its alias rather than the word.
+The feature is implemented by adding the alias of the command into its respective java document, and updating the AddressBookParser java document.
+The feature is implemented this way to utilise the command java documents that are available as opposed to creating a new java document for a certain alias. This feature is implemented to allow a user to enter a command without specifying the entire word of the command.
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: You need to add a + in the preceding paragraph to make this a new paragraph

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -385,6 +385,36 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
...
+
+=== Identifying a Duplicate feature [Planning to add]
+
+==== Current Implementation
+The CRM application will ensure that every lead in the address book is unique.
+The feature is implemented by checking the phone number and email address of a new lead with all of the phone numbers and email addressesof all leads and contacts in its respective hash sets in the add command. The feature will be implemented on the add command because the current application detects duplicates at the add command.
+However, the application identifies a duplicate based on whether all of the attributes of a lead is exactly the same as that of a lead/contact in the application.
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: Same as above, this is not a new paragraph

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -385,6 +385,36 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
...
+The phone number and email address of a lead will be checked in two respective hash sets.
+If any of these hash sets contain either the phone number of email address of a lead,
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: These are all not new paragraphs

**Comment by Sheikh-Umar** [comment]
Changes made based on feedback.

**Comment by davidten** [comment]
While generally I would disagree with the + on the blank lines, this passes for now

**Pull request merged**